src/main/resources/scripts/importer/cbioportalImporter.py

```diff
@@ -444,8 +444,13 @@ def check_dir(study_directory):
 def add_parser_args(parser):
     parser.add_argument('-s', '--study_directory', type=str, required=False,
                         help='Path to Study Directory')
+    parser.add_argument('-jvo', '--java_opts', type=str, required=False,
+                        help='Path to specify JAVA_OPTS for the importer. \
+                        (default: locates the jar path relative to the import script \
+                        and passes it as the JAVA_OPTS)')
     parser.add_argument('-jar', '--jar_path', type=str, required=False,
-                        help='Path to scripts JAR file')
+                        help='DEPRECATED ARGUMENT. Please, use -jvo/--java_opts instead. If you \
```

Comment:
Why not:
```
if args.jar_path:
    args.java_opts = f"-cp {args.jar_path} {args.java_opts}"
```
Then there is no 'need' to deprecate and you can replace it with something like 'WILL BE DEPRECATED IN THE FUTURE'?

Reply:
Hmm... but we want people to move away from using `jar_path`, right? We should then force them somehow to use the new attribute